apps/oc_chef_authz/src/oc_chef_policy_group.erl

```diff
@@ -0,0 +1,180 @@
+%% -*- erlang-indent-level: 4;indent-tabs-mode: nil; fill-column: 92 -*-
```

Comment:
Yep, let's just plan to do a sweep after to move the chef_object-type-things out of here and into chef_objects.  There are some other existing things that need moving as well.